### ISSUE TITLE
delphi tokyo

### ISSUE THREAD

**Issue by Thierry07** [yktoo/dklang#20]
Dklang is compatible with delphi 10.2.1 ?

i have a link error ( target ios11 ) 

  duplicate symbol ___rsrc_N6RCDATA13dklang_constsE in: 
Formain.o
[DCC Erreur fatale] F2588 Code d'erreur du lieur : 1 ($00000001)

And for windows 32 bits it's ok ( not for 64 )

**Comment by bjmillerpa** [comment]
I do not have 10.2.1 installed, so it has not been tested. If you can find a solution I will happily update the sources.

**Comment by Thierry07** [comment]
Ok, but i need to know which constant is concern by this error, any idea ?
What is the duplicated symbol ?

**Comment by bjmillerpa** [comment]
I doubt that it is a problem in the source. More likely some object code hanging around that shouldn't be. Try this: https://stackoverflow.com/questions/35666462/f2588-linker-error-code-1-00000001

**Comment by Thierry07** [comment]
Yes me too, i think that the problem is not in the source, i had already try to clean project and rebuild but the error is always here.
i try it in an another project and there is no error.

i’m afraid to have to redo all the project… ( it’s a lot of time lost )
